tools/loadtest/metrics/collect-metrics.sh

```
@@ -0,0 +1,1494 @@
+
+if jq -e '.rds_writer.cpu_utilization' "$OUTPUT" >/dev/null 2>&1; then
+  rds_w_cpu_avg=$(jq -r '.rds_writer.cpu_utilization.Average // "N/A"' "$OUTPUT")
+  rds_w_conns_avg=$(jq -r '.rds_writer.database_connections.Average // "N/A"' "$OUTPUT")
+  rds_w_deadlocks=$(jq -r '.rds_writer.deadlocks.Sum // "N/A"' "$OUTPUT")
+  printf "RDS Writer:    CPU=%s%%  Connections=%s  Deadlocks=%s\n" "$rds_w_cpu_avg" "$rds_w_conns_avg" "$rds_w_deadlocks"
+fi
+
+for reader_label in $(jq -r '.rds_readers[].instance // empty' "$OUTPUT" 2>/dev/null); do
+  rds_r_cpu_avg=$(jq -r --arg inst "$reader_label" '.rds_readers[] | select(.instance==$inst) | .cpu_utilization.Average // "N/A"' "$OUTPUT")
+  rds_r_conns_avg=$(jq -r --arg inst "$reader_label" '.rds_readers[] | select(.instance==$inst) | .database_connections.Average // "N/A"' "$OUTPUT")
+  rds_r_deadlocks=$(jq -r --arg inst "$reader_label" '.rds_readers[] | select(.instance==$inst) | .deadlocks.Sum // "N/A"' "$OUTPUT" 2>/dev/null || echo "N/A")
```

Shellcheck (0.11.0)

[warning] rds_r_deadlocks appears unused. Verify use (or export if used externally).

(SC2034)